scripts/superdesk-search/react.js

```diff
@@ -1551,17 +1551,37 @@
                         },
 
                         multiSelect: function(item, selected) {
```

Comment:
Would you please rename the parameter `item` to `items` please as now we are sending a list I guess...